include/swift/AST/DiagnosticsParse.def

```diff
@@ -653,6 +653,13 @@ ERROR(expected_rangle_protocol,PointsToFirstBadToken,
       "expected '>' to complete protocol composition type", ())
 ERROR(disallowed_protocol_composition,PointsToFirstBadToken,
       "protocol composition is neither allowed nor needed here", ())
+ERROR(any_composition_member,PointsToFirstBadToken,
+      "'Any' cannot be a member of a protocol composition", ())
+
+WARNING(deprecated_protocol_composition,PointsToFirstBadToken,
+        "'protocol<...>' composition syntax is deprecated; use infix '&' instead", ())
```

Comment:
I'm wondering if the message could be clearer for the majority of people (who don't bother with custom operators), who might not understand what is meant by "use infix & instead". It would be nice if we had an example, e.g. saying "P1 & P2".